mrp_multi_level/wizards/mrp_multi_level.py

```diff
@@ -200,45 +200,48 @@ def explode_action(
         mrp_date_demand = mrp_action_date
         if mrp_date_demand < date.today():
             mrp_date_demand = date.today()
-        if not product_mrp_area_id.product_id.bom_ids:
+        boms = self.env["mrp.bom"]
+        if product_mrp_area_id.supply_method == "buy":
+            boms = product_mrp_area_id.product_id.bom_ids.filtered(
+                lambda x: x.type == "subcontract"
```

Comment:
this is added in `mrp_subcontracting`, you should consider create a new glue module `mrp_multi_level_subcontracting`

Reply:
Done!